### ISSUE TITLE
attempt to compare number with nil

### ISSUE THREAD

**Issue by derekengel** [AvyAddons/AdiBags_Bound#2]
Getting this error using version 1.3.1

When opening the bag I get the following error

```
Message: Interface/AddOns/AdiBags_Bound/Main.lua:251: attempt to compare number with nil
Time: Mon Oct 31 17:23:03 2022
Count: 7
Stack: Interface/AddOns/AdiBags_Bound/Main.lua:251: attempt to compare number with nil
[string "@Interface/AddOns/AdiBags/core/Utility.lua"]:88: in function <Interface/AddOns/AdiBags/core/Utility.lua:84>
[string "=(tail call)"]: ?
[string "@Interface/AddOns/AdiBags/core/Filters.lua"]:146: in function Filter' [string "@Interface/AddOns/AdiBags/widgets/ContainerFrame.lua"]:789: in function FilterSlot'
[string "@Interface/AddOns/AdiBags/widgets/ContainerFrame.lua"]:823: in function DispatchItem' [string "@Interface/AddOns/AdiBags/widgets/ContainerFrame.lua"]:993: in function RedispatchAllItems'
[string "@Interface/AddOns/AdiBags/widgets/ContainerFrame.lua"]:1151: in function <Interface/AddOns/AdiBags/widgets/ContainerFrame.lua:1134>
[string "=(tail call)"]: ?
[string "@Interface/AddOns/AdiBags/widgets/ContainerFrame.lua"]:483: in function RefreshContents' [string "@Interface/AddOns/AdiBags/widgets/ContainerFrame.lua"]:468: in function ResumeUpdates'
[string "@Interface/AddOns/AdiBags/widgets/ContainerFrame.lua"]:451: in function <Interface/AddOns/AdiBags/widgets/ContainerFrame.lua:443>
[string "=[C]"]: in function Show' [string "@Interface/AddOns/AdiBags/core/Bags.lua"]:89: in function Open'
[string "@Interface/AddOns/AdiBags/core/Hooks.lua"]:112: in function <Interface/AddOns/AdiBags/core/Hooks.lua:109>
[string "=(tail call)"]: ?
[string "=(tail call)"]: ?
[string "OPENALLBAGS"]:1: in function <[string "OPENALLBAGS"]:1>

Locals:
```

**Comment by lucasvienna** [comment]
Looks like one of the items isn't returning its `bindType` properly. Would you mind giving me some more information?

Which WoW version are you on, Retail, Wrath, or Classic Era? Could I have a screenshot of your bags?

**Comment by derekengel** [comment]
I am using WoW Retail.  You're correct, it must have been an item.  I went to get the screenshot for you and unfortunately the error is not happening anymore. I went back to my bank and took a guess.  It looks like it's the Void-Scarred Beetle pet doing it.

![image](https://user-images.githubusercontent.com/8973148/199248725-907780c9-b70a-43f4-a437-4bc2732f31e2.png)

**Comment by lucasvienna** [comment]
Fixed in v1.3.2: https://github.com/lucasvienna/AdiBags_Bound/releases/tag/v1.3.2
